engine/classes/ElggUser.php

```diff
@@ -478,4 +477,20 @@ public function getPluginSetting(string $plugin_id, string $name, $default = nul
 		
 		return $this->psGetPluginSetting($plugin_id, $name, $default);
 	}
+	
+	/**
+	 * Notify the user about a given action on a subject
+	 *
+	 * @param string           $action  The action on $subject
+	 * @param \ElggData        $subject The notification subject
+	 * @param array            $params  Additional params
+	 *                                  use $params['methods_override'] to override the recipient notification methods (eg 'email' or 'site')
+	 * @param null|\ElggEntity $from    Sender of the message
+	 *
+	 * @return array
```

Comment:
document return value similar to elgg_notify_user

 Compound array of each delivery user/delivery method's success or failure.